tests/foreman/cli/test_host.py

```diff
@@ -2207,7 +2207,7 @@ def setUpClass(cls):
             u'lifecycle-environment-id': cls.env['id'],
             u'activationkey-id': cls.activation_key['id'],
             u'subscription': cls.subscription_name,
-        })
+        }, force_use_cdn=True)
```

Comment:
@abalakh this test use CDN to test subscriptions, nothing harmfull to use CDN here, and has to install some packages and to assert that that packages was installed.